tests/bin/identities/allowPermissionsDisallow.test.ts

```diff
@@ -0,0 +1,417 @@
+import type { Host, Port } from '@/network/types';
+import type { IdentityId, ProviderId } from '@/identities/types';
+import type { ClaimLinkIdentity } from '@/claims/types';
+import type { Gestalt } from '@/gestalts/types';
+import type { NodeId } from '@/nodes/types';
+import os from 'os';
+import path from 'path';
+import fs from 'fs';
+import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
+import PolykeyAgent from '@/PolykeyAgent';
+import { poll, sysexits } from '@/utils';
+import * as nodesUtils from '@/nodes/utils';
+import * as keysUtils from '@/keys/utils';
+import * as claimsUtils from '@/claims/utils';
+import * as identitiesUtils from '@/identities/utils';
+import * as testBinUtils from '../utils';
+import * as testUtils from '../../utils';
+import TestProvider from '../../identities/TestProvider';
+
+describe('allow/permissions/disallow', () => {
```

Comment:
This should be `allow/disallow/permissions`. And you want to focus on testing `allow` then `disallow` as exact opposites, and `permissions` command comes after both. Like allow 3, then disallow 2 and then permissions.

Reply:
Fixed up the order for this, as well as for trust/untrust/list (gestalts), set/unset/get (gestalt actions), and put/delete/get (tokens).